src/MicrosoftTeams.ts

```diff
@@ -627,6 +683,23 @@ namespace microsoftTeams {
     callbacks[messageId] = callback;
   }
 
+  /**
+   * Allows an app to retrieve information of all user joined teams
```

Comment:
Please add this to the top of this comment:
    * @private
    * Hide from docs.
    * ------